mtui/target/target.py

```diff
@@ -94,17 +78,11 @@ def connect(self) -> None:
             logger.warning(self._lock.locked_by_msg())
 
         # get system
-        self.system = self._parse_system()
+        self.system = parse_system(self.connection)
```

Comment:
and this the misspelled `self.connection` I mentioned above or `self.Connection`?

Reply:
above is misspelled :D nice catch